apps/site/pages/components/organisms/product-shelf.mdx

```diff
@@ -427,3 +364,11 @@ You can use [ProductCard variants](/components/molecules/product-card#variants)
 `data-fs-product-shelf-items`
 
 `data-fs-product-shelf-item`
+
+---
+
+## Best Practices
+
+### ❌ Don'ts
```

Comment:
Maybe add here about the reason to avoid the `ProductShelfItem` when using the `ProductShelf` with `Carousel` too, idk. It's up to you 🤝

Reply:
Nice! Added [here](https://github.com/vtex/faststore/pull/1735/commits/22763a5a5913c769bfe5658ad13fcc255f394440)